handlers/handlers_test.go

```diff
@@ -300,7 +300,7 @@ func TestUnitReadFailure(t *testing.T) {
 				So(w.Code, ShouldEqual, http.StatusBadRequest)
 
 				So(len(mockedRendererClient.BuildPageCalls()), ShouldEqual, 0)
-				So(len(mockedSearchClient.GetSearchCalls()), ShouldEqual, 1)
+				So(len(mockedSearchClient.GetSearchCalls()), ShouldEqual, 2)
```

Comment:
It took me a while to understand why this had to be increased to 2. Could we add some assertions to ensure they are called with the right params (it looks like one of them should not contain "content_type" nor "topics")?
It would make more explicit what the 2 calls are for